### PR TITLE
Added Artron Inhibitor

### PR DESCRIPTION
**Pull request by NovaAstral**
- Added Artron Inhibitor, an entity like the Time Distortion Generator, except it will slowly drain the Artron Energy from any TARDIS within its radius

Note: Related Sonic Screwdriver PR: https://github.com/MattJeanes/Sonic-Screwdriver/pull/41

### CODE REVIEW

File: lua/entities/gmod_artron_inhibitor/shared.lua [hunk 1]
```diff
@@ -0,0 +1,17 @@
+-- Artron Inhibitor by Nova Astral, an edit of the Time Distortion Generator by parar020100 and JEREDEK
+
+ENT.Type = "anim"
+ENT.Base = "base_gmodentity"
+
+ENT.PrintName = "#TARDIS.ArtronInhibitor"
```
Comment (AmyJeanes) [inline lua/entities/gmod_artron_inhibitor/shared.lua:6]: There are missing translations for the following strings
- `ArtronInhibitor`
- `ArtronInhibitor.Instructions`
- `ArtronInhibitor.Starting`
- `ArtronInhibitor.Disabled`
- `ArtronInhibitor.Enabled`

These should be added into the `i18n/en.json` file and run the Generate Languages task in VSCode (Ctrl+Shift+P -> Run task -> Generate Languages)

Note you need PowerShell Core installed for this to work

File: lua/entities/gmod_artron_inhibitor/init.lua [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+    if self.FlyTime ~= nil and CurTime() - self.FlyTime > 3 then
+        self.FlyTime = nil
+        self.On = true
+        self:SetEnabled(true)
+
+        TARDIS:Message(self.LastActivator, "ArtronInhibitor.Enabled")
+        self.LastActivator = nil
+
+        self:GetPhysicsObject():SetVelocity(Vector(0, 0, 0))
+        self:SetColor(Color(164, 90, 250, self:GetColor().a))
```
Comment (AmyJeanes) [inline lua/entities/gmod_artron_inhibitor/init.lua]: It would make sense to have this be a different colour to the TDG

File: lua/entities/gmod_artron_inhibitor/init.lua [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+    return self.Radius
+end
+
+function ENT:SetRadius(radius)
+    self.Radius = math.Clamp(radius,0,2048)
+    self:TriggerWire("Radius",self.Radius)
+    return self.Radius
+end
+
+function ENT:Repair(repair)
```
Comment (AmyJeanes) [inline lua/entities/gmod_artron_inhibitor/init.lua:58]: We would also want to add this into the sonic screwdriver so it can be repaired etc, can you make another PR for that?

I've added you as a collaborator to that repo now

**Review by AmyJeanes — CHANGES_REQUESTED** [review]
This looks like a useful roleplay tool, nice work! We're missing translations and a couple other tiny things but looks good to me overall.

File: lua/entities/gmod_artron_inhibitor/init.lua (hunk 3 above)
Comment (NovaAstral) [reply lua/entities/gmod_artron_inhibitor/init.lua:58]: made a pr in the sonicsd for this

File: lua/entities/gmod_artron_inhibitor/init.lua (hunk 2 above)
Comment (NovaAstral) [reply lua/entities/gmod_artron_inhibitor/init.lua]: started color is now orange (the startup color of the tdg because annoyingly its a good color for artron energy)
startup color is blue

File: lua/entities/gmod_artron_inhibitor/shared.lua (hunk 1 above)
Comment (NovaAstral) [reply lua/entities/gmod_artron_inhibitor/shared.lua:6]: Added language but I don't have PowerShell Core and I don't know which one is the correct one.

Comment (AmyJeanes) [reply lua/entities/gmod_artron_inhibitor/shared.lua:6]: Looks good! Note I've made some small tweaks to match the TDG language

**Review by AmyJeanes — APPROVED** [review]
Looks good from my perspective, but I also want @parar020100 to have a look at this as well

**Review by parar020100 — APPROVED** [review]
Looks good to me, nice job!
